scripts/daily_plot.py

```diff
@@ -70,9 +75,14 @@ def create_plot(df_plt_today, now, is_top=None):
 
     df_plt_selection_today = df_plt_today[df_plt_today.Com_Name.isin(plt_selection_today.index)]
 
+    conf = get_settings()
+
     # Set up plot axes and titles
     height = max(readings / 3, 0) + 1.06
-    f, axs = plt.subplots(1, 2, figsize=(10, height), gridspec_kw=dict(width_ratios=[3, 6]), facecolor='#77C487')
+    if conf['COLOR_SCHEME'] == "dark":
+        f, axs = plt.subplots(1, 2, figsize=(10, height), gridspec_kw=dict(width_ratios=[3, 6]), facecolor='darkgrey')
+    else:
+        f, axs = plt.subplots(1, 2, figsize=(10, height), gridspec_kw=dict(width_ratios=[3, 6]), facecolor='none')
```

Comment:
```suggestion
    if conf['COLOR_SCHEME'] == "dark":
        facecolor='darkgrey'
    else:
        facecolor='none'
    f, axs = plt.subplots(1, 2, figsize=(10, height), gridspec_kw=dict(width_ratios=[3, 6]), facecolor='darkgrey')
```

Reply:
Change made using variable instead of hardcoded color